docs/RFID.md

```diff
@@ -21,8 +21,10 @@ The PhidgetRFID library makes for intuitive and lightning fast development witho
 |LEDon|bool|yes|LED state  : ` 1 ` is On, ` 0 ` is Off|
 |lastTag|number|no|` ID ` of the last detected tag|
 |tagState|bool|no|RFID tag present : ` 1 ` is Present, ` 0 ` is Not Present|
-|tag2|object|no|Tag info if present ` tag protocol ` + ` tag ID `|
-|tagLoss2|object|no|` tag protocol ` + ` tag ID ` of the last detected tag|
+|tag2.protocol|number|no|Protocol info number|
```

Comment:
protocol info number. I would say number is not needed since the type is number. But whatevs